Trivago/Jade/Application/Framework/JadeBundle/Controller/JsonApiController.php

```diff
@@ -475,12 +476,12 @@ protected function getRepository($resourceName)
     }
 
     /**
-     * @param string $resourceName
-     * @param string $action
+     * @param string        $resourceName
```

Comment:
This formatting is not respecting the coding style used in the rest of the library